source/basic.tex

```diff
@@ -4,11 +4,11 @@
 \gramSec[gram.basic]{Basic concepts}
 
 \pnum
-\begin{note} This Clause presents the basic concepts of the \Cpp language.
+\begin{note} This Clause presents the basic concepts of the \Cpp{} language.
 It explains the difference between an object and a
 name and how they relate to the value categories for expressions.
 It introduces the concepts of a
-declaration and a definition and presents \Cpp's
+declaration and a definition and presents \Cpp{}'s
```

Comment:
Just FYI: I don't think the braces are required here (unless you want to prevent kerning).

Reply:
The idea is to have a general rule that all \Cpp macros are followed by {}. Whether that's necessary in a particular situation should not matter.

Reply:
The only downsides are the adverse impact on readability of the source and (potentially) kerning.

I doubt that kerning is much of a problem here. And giving up a little bit of readability to gain consistency (for those editors who are less familiar with TeX's parsing rules) sounds like a reasonable thing to do.

(We could also add a lint check to warn in suspicious uses of `\Cpp` without the terminating `{}`.)

Reply:
This change does in fact not cause any visual diff.

@godbyk: Yes, sure, that sounds useful!